scripts/globals/abilities/chocobo_jig_ii.lua

```diff
@@ -20,6 +20,11 @@ abilityObject.onUseAbility = function(player, target, ability)
     local durationMultiplier = 1.0 + utils.clamp(player:getMod(xi.mod.JIG_DURATION), 0, 50) / 100
     local finalDuration      = math.floor(baseDuration * durationMultiplier)
 
+    -- TODO: Chocobo Jig will not override all types of weight effect. Find out which aren't overriden.
+    if target:hasStatusEffect(xi.effect.WEIGHT) then
```

Comment:
General question, do we need to check for a status before we try and remove it, or is removal of a status that isn't there a safe no-op?

Reply:
Generaly it isnt needed.

However, since I KNOW there are certain cases where Weight effect isn't removed (probably based on effect power) I decided to keep the check in there, since I know me, or some1 else, will end up fetching the effect power and removing it based on it.